llvm/test/Transforms/LoopVectorize/RISCV/evl-compatible-loops.ll

```diff
@@ -0,0 +1,262 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -force-tail-folding-style=data-with-evl \
+; RUN: -prefer-predicate-over-epilogue=predicate-dont-vectorize \
+; RUN: -force-target-instruction-cost=1 -mtriple=riscv64 -mattr=+v -S < %s | FileCheck %s
```

Comment:
why is `-force-target-instruction-cost=1` needed?

Reply:
It was required for the third case to be profitable. It can be removed now since the case is redundant.